ledger/participant-integration-api/src/main/scala/platform/apiserver/ErrorCodesVersionSwitcher.scala

```diff
@@ -5,12 +5,18 @@ package com.daml.platform.apiserver
 
 import io.grpc.StatusRuntimeException
 
-final class ErrorCodesVersionSwitcher(enableErrorCodesV2: Boolean) {
+/** A mechanism to switch between the previous error codes (v1) and the new self-service error codes (v2).
```

Comment:
```suggestion
/** A mechanism to switch between the legacy error codes (v1) and the new self-service error codes (v2).
```

Reply:
Ok


I wonder if you could avoid "legacy" 

> legacy
> If possible, use a more precise term. If you do use legacy, include or point to a definition to clarify what you mean in the current context. Do not use legacy with any sort of pejorative connotation.
(https://developers.google.com/style/word-list#letter-l)

Reply:
Huh, I didn't know it has potentially negative connotation